coalib/bears/Bear.py

```diff
@@ -249,8 +249,14 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            if self.kind():
```

Comment:
Yeah, you are right.. I just assumed that BEAR_KIND.GLOBAL_BEAR would return something more informative, but it seems that we are using a it as a binary `0` is Local and `1` is global.

But I still suggest to use `self.kind() == BEAR_KIND.GLOBAL` as it is more readable and understandable then `if self.kind()`..

@Makman2 would you suggest the same ? ^^